LabelPoint[] points
float32 scaling = 0.05
float32 start_z = 0.5
int32 rating = 800
bool showMarker = 0